src/site/content/en/learn/css/spacing/index.md

```diff
@@ -32,15 +32,15 @@ each with its own strengths and caveats.
 ## HTML spacing
 
 HTML itself provides some methods to space elements.
-The `<br />` and `<hr />` elements allow you to space elements in the block direction,
+The `<br>` and `<hr>` elements allow you to space elements in the block direction,
 which if you are in a latin-based language,
 is top-to-bottom.
 
-If you use a `<br />` element,
+If you use a `<br>` element,
 it will create a line-break,
 just like if you were to press your enter key in a word processor.
 
-The `<hr />` creates a horizontal line with space either-side, known as `margin`.
+The `<hr>` creates a horizontal line with space either-side, known as `margin`.
```

Comment:
@rachelandrew have a recommendation here for style?

Seems like a good topic/commit for a separate PR since the choice may effect other documents and codepens?

Reply:
I thought I checked the other files of the course in the IDE for other matches of `/>`, before opening this pull request, however it seems like I missed two:

* the `<img />` tag in the very [next chapter](https://web.dev/learn/css/pseudo-elements/#::before-and-::after)
  https://github.com/GoogleChrome/web.dev/blob/59896013fd123c437b866220ddfa110b093b27a5/src/site/content/en/learn/css/pseudo-elements/index.md#L86
* the `<link />` tag in the [quiz about the cascade](https://web.dev/learn/css/the-cascade/#position-and-order-of-appearance)
  https://github.com/GoogleChrome/web.dev/blob/59896013fd123c437b866220ddfa110b093b27a5/src/site/content/en/learn/css/the-cascade/position.assess.yml#L13

This is how I looked for the occurrences:
```sh
find src/site/content/en/learn/css \( -name "*.md" -o -name "*.yml" \) -print0 | xargs -0 grep "/>"
```

However, looking through the CodePens might be a little more involved.